playground/test-utils.ts

```diff
@@ -32,14 +32,13 @@ export const ports = {
   ssr: 9600,
   'ssr-deps': 9601,
   'ssr-html': 9602,
+  'ssr-hmr': 9609, // not imported but used in `ssr-hmr/vite.config.js`
   'ssr-noexternal': 9603,
   'ssr-pug': 9604,
   'ssr-webworker': 9605,
-  'proxy-bypass': 9606, // not imported but used in `proxy-hmr/vite.config.js`
-  'proxy-bypass/non-existent-app': 9607, // not imported but used in `proxy-hmr/other-app/vite.config.js`
```

Comment:
Why were these removed? The proxy-bypass playground wasn't touched, no? I did a PR to fix a CI issue recently https://github.com/vitejs/vite/pull/15716/files#diff-9630490639a9d111584f0f238575fd2e3c473f4b06bbf12482702b2e9f6864d1R38, it is strange we don't get a conflict against main here. Maybe we should merge main again.

Reply:
We don't get conflicts because this is how conflicts were resolved (incorrectly) :P